### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD005.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#9] [General]
# https://blog.3place.cn/3place/fsd/FSD005.html

https://blog.3place.cn/3place/fsd/FSD005.html

**Comment by picklecai** [comment]
他肯定会等在那棵白杨树旁。

东门之杨，其叶牂牂。东门之杨，其叶肺肺。

**Comment by picklecai** [comment]
衰弱不堪的年老的严冬，
已经退藏到荒凉的山中。
它一面逃跑，一面还送来
无力的阵雨，夹杂着冰雹，
斑斓地掩复葱绿的原野；

好可怜的冬天，作了最后的挣扎还要惹人嘲笑。
谁又肯一点不作挣扎、引颈就戮呢？
总有衰弱的时候，
不堪，那就是撑不住了。
撑不住的时候还作希望的挣扎，
那时多么强烈的企图心！
这样的强烈，不该被嘲笑。

**Comment by picklecai** [comment]
她要让万物多彩而生动；
可是这一带还不见花开，
就唤盛装的人群来替代。

人类眼里，人的生动最可爱，退而求其次是生命的活力最可爱。“生“才有“动”，“活”才有“力”。
人的一生，大概都在维持、保护这“生”、这“活”。

**Comment by picklecai** [comment]
瞧大家都在活泼地散步，
分头穿过了田野和菜园，
在河面上上下下的各处，
漂荡着好多快乐的小船，
最后的一只，满载得快要
沉下去似地才离开河岸。

参差荇菜，左右采之。
参差荇菜，左右芼之。

怪不得，人需要度假。

**Comment by picklecai** [comment]
我敬你一杯，高声祝愿，
它不仅解除你的口渴，
还愿照它含有的滴数，
增加先生同样的岁数。

这个敬酒词，绝了

**Reply by picklecai** [reply]
这个词，非常地王熙凤。

**Comment by picklecai** [comment]
他匆匆地赶到那里，
就在那时他的手拐子
撞着一位女多娇；

这个“他”是谁？浮士德还是瓦格纳？

**Comment by picklecai** [comment]
目前为止，浮士德：
博学、
好奇心/求知欲旺盛（为了求知可以不择手段）、
尊重自然、
喜爱生命、
对穷苦人富有爱心。

我在想为什么选择了他。后面继续看。

**Comment by picklecai** [comment]
受这许多人尊敬，哦，伟大的人，
不知你心中发生什么感慨！
谁能凭着自己的高才
而如此受惠，真是福人！

瓦格纳好羡慕，可见瓦格纳肯定做不到“名利皆空”。

**Comment by picklecai** [comment]
我常独坐在那里沉思默想，
折磨自己，进行斋戒和祷告。

这个折磨，估计翻译有误。估计是苦行僧那一套，不知道该用什么词表达比较好。直接用“鞭笞”？

**Comment by picklecai** [comment]
我想强求在天之主
把那一场瘟疫扑灭。
群众的赞扬对我简直像讥刺。
但愿你能看透我的内心。

我就说和瓦格纳想法大不相同。
这是“仁”。

**Reply by picklecai** [reply]
我们就这样使用恐怖的灵丹，
在这群山万壑之间，
猖狂肆虐，比瘟疫更猛。
我亲自把这件礼物赠给成千的人士，
他们凋零了，我却要在世
听人赞扬无耻的元凶。

原来是为了这个而觉得“讽刺”。实在

**Comment by picklecai** [comment]
在这群山万壑之间，
猖狂肆虐，比瘟疫更猛。

子曰：苛政猛于虎。

**Comment by picklecai** [comment]
我们不知者，正合我们所用，
我们所知者，却没有用处。

书到用时方恨少。

**Reply by picklecai** [reply]
“不知者，正合我们所用”，也可以用来描述做得出来、却缺少合适的表达力的情况，或者：并不完全懂，但照葫芦画瓢抄作业成功。

**Reply by picklecai** [reply]
1. 书到用时方恨少，库里没有应对当前问题的办法。
2. 会做不会说，缺少表达力。会说的，却不会做，只是纸上谈兵。
3. 照葫芦画瓢，抄作业成功。但不知其所以然。很擅长的，却不对应当前问题，于事无补。 ​​​

**Comment by picklecai** [comment]
你瞧，在夕阳掩映之下，
绿裹的农家蓬荜生辉。

绿树村边合，青山郭外斜。

**Comment by picklecai** [comment]
太阳隐退了，一天就此告终，
她奔向彼方，开拓新的生涯。
啊，但愿我能插翅高飞凌空，
永远不停地追随着她！

夸父

**Comment by picklecai** [comment]
看我脚下静静的人世
熠熠辉映着永恒的斜阳，
群山发出红光，溪谷一片安谧，
银色的小溪流入金色的大江。

心随太阳去了，也就开阔了

**Comment by picklecai** [comment]
我已经听到村民的喧嚷，
这是民众的真正的天堂，
不论老和少都欣然欢腾：
这里我是人，我能做个人！

在象牙塔时，不是人，是什么？
只有在人堆里才感觉到是人吗？
人群的什么感染到了这种“人”的感觉？
从前我独处时，常常惶然，感觉不到周围有哪怕一点点生机。浮士德也是这样吗？

**Comment by picklecai** [comment]
你何必为了此事烦恼！
施行传授来的技术，
问心无愧，精确无误，
这种大丈夫行为岂不够好？

瓦格纳的意思是：抄作业不用负责任。

**Comment by picklecai** [comment]
头上是太空，脚下是一片海波。
一场好梦！

这样的梦，既波澜壮阔又不安定，得心飞扬起来才hold得住。待在象牙塔里无法做出这样的梦——除非压力太大！

**Comment by picklecai** [comment]
一个怀着一种强烈的情欲，
以它的卷须紧紧攀附着现世；
另一个却拚命地要脱离尘俗，
高飞到崇高的先辈的居地。

我突然觉得这两个小人儿可以合二为一，大概因为我没有品尝过现世之美酒。

**Comment by picklecai** [comment]
他们爱窃听，惯于幸灾乐祸，
他们爱服从，因为要欺骗我们；
他们装腔作势，像来自天国，
说谎时像天使一样轻声。

这也适用于这句话：最了解你/让你感到舒服的人，一是爱人，二是骗子。

**Comment by picklecai** [comment]
这到底是狗还是精灵？印证了那句话：心里有什么，眼里才能见到什么。心里认为是普通的狗，怎么看都是普通的狗。心里认为是精灵，怎么看都有异常之处。
